ibis/expr/rewrites.py

```diff
@@ -0,0 +1,60 @@
+"""Some common rewrite functions to be shared between backends."""
+from __future__ import annotations
+
+import functools
+from collections.abc import Mapping
+
+import ibis.expr.datatypes as dt
+import ibis.expr.operations as ops
+from ibis.common.patterns import pattern, replace
+from ibis.util import Namespace
+
+p = Namespace(pattern, module=ops)
+
+
+@replace(p.FillNa)
+def rewrite_fillna(_):
```

Comment:
I thought about using a more verbose name for these, but none felt right. `desugar_fillna` or `simplify_fillna` might be better, but :shrug:.